.github/workflows/update_tag1_d7es.yml

```diff
@@ -0,0 +1,24 @@
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Runs daily at midnight UTC
+  workflow_dispatch:
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  check-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Set up dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y jq wget curl
+      - name: Check and Update Module
+        run: .github/update_tag1_d7es.sh
```

Comment:
idk if you have tested this, but I expect this won't trigger the build (GHA, at least) on the newly created PR. Is that ok?

Reply:
This is meant to run on cron, not PR, that's correct.